src/main/java/com/alkemy/ong/config/segurity/SecurityConfig.java

```diff
@@ -79,6 +79,8 @@ protected void configure(HttpSecurity http) throws Exception {
         .hasRole(RoleType.ADMIN.name())
         .antMatchers(HttpMethod.POST, "/members")
         .hasRole(RoleType.USER.name())
+        .antMatchers(HttpMethod.GET, "/categories/*")
```

Comment:
Este patrón hace que /categories/1 como /categories/1/news o cualquier otra cosa... se accesible por el role seleccionado en la línea 83. El patrón correcto para este endpoint sería: `"/categories/{\\d+}"`

Reply:
No esta funcionando colocando como lo comentaste Maga :"(

Reply:
Que raro... debería funcionar. Fijate si lo podes revisar con algún compañero, utilizado de Rubber Duck, en caso de que se te este escapando algo. Aveces uno se olvida algún que otro detallin si estuvo mucho rato mirando el mismo código.

Ref: https://kenzie.snhu.edu/blog/what-is-rubber-duck-debugging/